WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt

```diff
@@ -137,6 +146,13 @@ sealed class PostListItemViewHolder(
         menu.show()
     }
 
+    private fun getDrawableFromAttribute(context: Context, attributeId: Int): Drawable? {
```

Comment:
NitPick: We might want to consider moving this method into a different class (eg. ViewUtils.kt). Wdyt?

Reply:
Moved to a new file `ContextUtils.kt` in f6e2cbb27a391e23b4de7aff73c143517ea40031 - honestly not sure this is the right place to move it.

Reply:
I think it's definitely an improvement over having it in the VH. We can move it from the ContextUtils to another file if we come up with a more appropriate place.